Add --static support to libart2-config --libs to avoid -lm
Debian #375812, GNOME #531278

Index: b/libart-config.in
===================================================================
--- a/libart-config.in	2009-04-18 14:29:25.000000000 +0200
+++ b/libart-config.in	2012-06-17 17:37:35.925545343 +0200
@@ -3,6 +3,8 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 exec_prefix_set=no
+libs=""
+output_libs=no
 
 usage="\
 Usage: libart-config [--prefix[=DIR]] [--exec-prefix[=DIR]] [--version] [--libs] [--cflags]"
@@ -43,8 +45,11 @@
       echo $includes
       ;;
     --libs)
-      libdirs=-L@libdir@
-      echo $libdirs -lart_lgpl_2 -lm
+      libs="$libs -L@libdir@ -lart_lgpl_2"
+      output_libs=yes
+      ;;
+    --static)
+      libs="$libs -lm"
       ;;
     *)
       echo "${usage}" 1>&2
@@ -54,3 +59,7 @@
   shift
 done
 
+if test $output_libs = yes ; then
+    echo $libs
+fi
+
